lib/splitio-api-binding.js

```diff
@@ -85,8 +85,9 @@ class SplitioApiBinding {
     const { allChanges, since } = await this.getAllChanges({ path })
     allChanges.forEach(change => {
       change.splits.forEach(split => {
-        if (split.status !== 'ARCHIVED') {
-          splits[split.name] = split
+        splits[split.name] = split
```

Comment:
i think this is clearer because you wouldn't expect to add something to a map and them remove it immediately after:

```js
if (split.status === 'ARCHIVED') {
  delete splits[split.name]
} else {
  splits[split.name] = split
}
```